### DISCUSSION TITLE
Linear and logistic regression video explainers

### DISCUSSION THREAD

**Discussion by bstollnitz** [community/community#61884] [Programming Help]
Linear and logistic regression are basic concepts for AI practiticioners. If you need to brush up on these topics, here are two videos I recorded (as part of a longer series):
Linear regression: https://www.youtube.com/watch?v=CRxFT8oTDMg&list=PL5ixj5_VDwLaHcsZTqowCnhQUDMATV6Iv&index=14&t=2s
Logistic regression:  https://www.youtube.com/watch?v=KpeCT6nEpBY&list=PL5ixj5_VDwLaHcsZTqowCnhQUDMATV6Iv&index=22&t=2s

Enjoy!

**Comment by IgnacioPalma** [comment]
This is amazing! :exploding_head: :heart:

**Reply by bstollnitz** [reply]
Thank you so much Ignacio!
